apps/expo-go/android/build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
     gradleDownloadTaskVersion = '5.0.1'
     repositoryUrl = "file:${System.env.HOME}/.m2/repository/"
 
-    ndkVersion = "26.3.11579264"
+    ndkVersion = "26.1.10909125"
```

Comment:
Downgrading seems weird. Do you know why we used `26.3` before?

Reply:
I think you needed to bump it in #29816 because expo go would not build at the time. It wouldn't build for me using `26.3` but `26.1` it was RN core is using so now we are aligned again.